### PR TITLE
docs: change directory structure, overview content in each section an…

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
Changes
- Add usage section for CLI and Kubernetes
- Rewrite existing overview sections -- namely, for events and policies
- Add Overview page to docs section where there was none
- Resolve broken links across docs

TODO:
- Add Kubernetes Operator Usage Guide by @josedonizetti

### CODE REVIEW

File: docs/docs/deep-dive/caching-events.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ sudo ./dist/tracee \
     -o format:json \
     -o option:parse-arguments \
     -trace container \
-    --cri docker:/var/run/docker.sock
+    --crs docker:/var/run/docker.sock
```
Comment (geyslan) [inline docs/docs/deep-dive/caching-events.md:47]: Did you brought main upstream onto your branch? The crs->cri is a recent change.

Comment (AnaisUrlichs) [reply docs/docs/deep-dive/caching-events.md]: yes, it got a bit messed up but I think I fixed it in my PR

File: mkdocs.yml [hunk 2]
```diff
@@ -635,4 +633,4 @@ extra:
 
 plugins:
     - search
-    - macros
+    - macros
```
Comment (geyslan) [inline mkdocs.yml:636]: nit: add new line at the end.

image

File: docs/docs/events/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* syscalls 
+* network 
+* security 
+* lsm 
+* containers 
+* misc
+
+This section documents all of the different events that Tracee exposes.
+
+## Defining events
```
Comment (itaysk) [inline docs/docs/events/index.md:16]: Defining events sounds like adding new events to Tracee. maybe change to "selecting events"

File: docs/docs/install/config/kubernetes.md [hunk 4]
```diff
@@ -17,22 +17,20 @@ metadata:
 data:
   config.yaml: |-
     cache:
-        type: mem
-        size: 512
+      - cache-type=mem
```
Comment (geyslan) [inline docs/docs/install/config/kubernetes.md:20]: These must be the new structured.

File: docs/docs/events/index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+Tracing the `execve` events in a [policy](../policies/index.md):
```
Comment (itaysk) [inline docs/docs/events/index.md:18]: I think it needs an introduction "Events are selected through a policy etc..."

File: docs/docs/flags/containers.1.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
-title: TRACEE-CRI
+title: TRACEE-CRS
```
Comment (geyslan) [inline docs/docs/flags/containers.1.md:2]: Must be CRI.

File: docs/docs/flags/containers.1.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
...
 section: 1
-header: Tracee CRI Flag Manual
+header: Tracee CRS Flag Manual
```
Comment (geyslan) [inline docs/docs/flags/containers.1.md:4]: ditto.

File: docs/docs/events/index.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+	annotations:
+		description: traces execve events
+spec:
+	scope:
+	  - global
+	rules:
+	  - event: execve
+```
+
+If no event is passed with [filters] or [policies], tracee will start with a set of default events.
```
Comment (itaysk) [inline docs/docs/events/index.md:34]: Not sure it's correct. I think it should be "if no policy is defined, tracee will start with.."

File: docs/docs/events/index.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+Please head over to the [Tracee usage](../usage/index.md) documentation for more information on configuring events.
```
Comment (itaysk) [inline docs/docs/events/index.md:36]: broken link?

File: docs/docs/events/index.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+### Event Sets QUESTION
```
Comment (itaysk) [inline docs/docs/events/index.md:38]: I suppose QUESTION is you note to self. I'm also not sure about this

File: docs/docs/events/index.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+	annotations:
+		description: traces execve events
+spec:
+	scope:
+	  - global
+	rules:
+	  - event: syscalls
+```
+
+## Read in AVD
```
Comment (itaysk) [inline docs/docs/events/index.md:57]: we can remove this now that all events are documented here and actually AVD is out of date

File: docs/docs/events/index.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
+# Events
+
+Events refer to the activity in a system that tracee monitors. There are two main types of events, built-in events that are part of Tracee and custom events, which references the way users define other events that Tracee should monitor.
```
Comment (itaysk) [inline docs/docs/events/index.md:3]: activity in a system -> system activity
two main types -> two types
references the way users define other events that Tracee should monitor -> is user defined events

File: docs/docs/install/config/cli.md [hunk 13]
```diff
@@ -1,4 +1,4 @@
-# Config
+# CLI Config
```
Comment (itaysk) [inline docs/docs/install/config/cli.md:1]: Configuring Tracee through the CLI

File: docs/docs/install/config/cli.md [hunk 14]
```diff
@@ -1,4 +1,4 @@
...
 
 ## Configuring Tracee with the `--config` Flag
```
Comment (itaysk) [inline docs/docs/install/config/cli.md:3]: redundant header

File: docs/docs/events/index.md (hunk 10 above)
Comment (AnaisUrlichs) [reply docs/docs/events/index.md:38]: Ah yes, something I need to check with the team

File: docs/docs/policies/rules.md [hunk 15]
```diff
@@ -1,16 +1,13 @@
 # Rules
 
-Rules determine which events a policy should trace. 
+Rules determine which events a policy should trace and make it possible to narrow down when and event should be recorded.
```
[the remark this answers is not in the document]
Comment (AnaisUrlichs) [reply docs/docs/policies/rules.md:3]: I rewrote the section

File: docs/docs/outputs/index.md
[the remark this answers is not in the document]
Comment (AnaisUrlichs) [reply docs/docs/outputs/index.md:1]: Yes, working on that next

File: docs/docs/install/config/kubernetes.md (hunk 4 above)
Comment (AnaisUrlichs) [reply docs/docs/install/config/kubernetes.md:20]: but that is in your PR, no?

Comment (geyslan) [reply docs/docs/install/config/kubernetes.md:20]: Mine (structured config) was already merged. So you're rolling it back hehehe.

**Comment by itaysk** [comment]
For a future PR

the install/config section should probably have an index.md that shows the different configuration options (irrespective of how to apply it which is in the cli/kuberntes pages).

**Pull request merged**